### PR TITLE
[user/reservation/testPDF] - PDF function

### PR DESCRIPTION
**Pull request by Tomo-Furu**
At the moment, there is only a UI for PDF output. 
The backend, such as connection to the database, will be developed later.

Added:
- Install barryvdh/laravel-dompdf
- Added PDF controller
- Added pdf_download.blade.php & pdf_view.blade.php
- Added 2 of Route
- Added route in list.blade.php

### CODE REVIEW

File: app/Http/Controllers/PDFController.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use Barryvdh\DomPDF\Facade\Pdf;
+
+class PDFController extends Controller
+{
+
+    public function pdf_generator_get(Request $request)
```
Comment (kredo-school) [inline app/Http/Controllers/PDFController.php]: You're not using $request, consider removing this.

File: app/Http/Controllers/ReservationController.php [hunk 2]
```diff
@@ -42,6 +55,19 @@ public function showCompletionReservation()
...
             ->with('confirmedReservations', $confirmedReservations)
-            ->with ('userAttribute', $userAttribute);
+            ->with('userAttribute', $userAttribute);
     }
+
+
+    public function pdf()
+    {
+        if (Auth::check()) {
+            $reservations = Reservation::with('user')->get();
```
Comment (kredo-school) [inline app/Http/Controllers/ReservationController.php]: Are you gonna pass the $reservations in the view ? if not, consider removing it

File: app/Http/Controllers/PDFController.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        $users = User::get();
+        $userNames = $users->pluck('name');
+        $data = [
+            'title' => 'PDF',
```
Comment (keanbillyfernandez) [inline app/Http/Controllers/PDFController.php]: Please make this more friendly. You can try "PDF[name][date]"

File: app/Http/Controllers/PDFController.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
```
Comment (keanbillyfernandez) [inline app/Http/Controllers/PDFController.php]: Also remove this if you are not using the $request

File: app/Http/Controllers/PDFController.php (hunk 1 above)
Comment (YoshimiHELLo) [reply app/Http/Controllers/PDFController.php]: I've removed it

File: app/Http/Controllers/ReservationController.php (hunk 2 above)
Comment (YoshimiHELLo) [reply app/Http/Controllers/ReservationController.php]: I've removed it

File: app/Http/Controllers/PDFController.php (hunk 3 above)
Comment (YoshimiHELLo) [reply app/Http/Controllers/PDFController.php]: They are no need  so I've removed them

Comment (YoshimiHELLo) [reply app/Http/Controllers/PDFController.php:17]: I've removed it

File: app/Http/Controllers/PDFController.php (hunk 4 above)
Comment (YoshimiHELLo) [reply app/Http/Controllers/PDFController.php]: I've removed it

**Pull request merged**